### PR TITLE
Concurrent OpenShift Shenanigans

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

A workflow demonstrating concurrent shenanigans in an OpenShift cluster.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: advanced-examples/system-performance/concurrent-k8s/config.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    level: debug
+deployers:
+  image:
+    deployer_name: podman
+    deployment:
+      imagePullPolicy: IfNotPresent
+  python:
+    deployer_name: python
+    modulePullPolicy: Always
+    pythonPath: /home/mleader/.cache/pypoetry/virtualenvs/arcaflow-plugin-kill-pod-2rBSPLV3-py3.9/bin/python
```
Comment (dbutenhof) [inline advanced-examples/system-performance/concurrent-k8s/config.yaml:14]: This probably needs to be changed. 😆

File: advanced-examples/system-performance/concurrent-k8s/config.yaml
Comment (dbutenhof) [inline advanced-examples/system-performance/concurrent-k8s/config.yaml:1]: Ending files with a newline is always appreciated. I'm unamused by GitHub's complaint icon.

File: advanced-examples/system-performance/concurrent-k8s/input.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+kubeburner_list:
+  -  kubeconfig_path: '/home/mleader/.kube/config'
```
Comment (dbutenhof) [inline advanced-examples/system-performance/concurrent-k8s/input.yaml:2]: Path?

**Review by dbutenhof — COMMENTED** [review]
I probably should have ignored this for now! Your `/home/mleader` paths probably need some work before pushing a "real" draft?

File: advanced-examples/system-performance/concurrent-k8s/config.yaml (hunk 1 above)
Comment (mfleader) [reply advanced-examples/system-performance/concurrent-k8s/config.yaml:14]: Oh 100%

**Comment by mfleader** [comment]
This work is not necessary at this time without the kube burner and kill pod plugins in the workflow. Currently, we want these workflows to only use Arcaflow plugin's that are owned by the Arcalot organization. Neither the kube burner nor the kill pod plugin are owned by the Arcalot organization.

**Pull request closed without merging**